tests/test.d

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env rdmd
+/**
+Platform independent port of `test.sh`. Runs the tests in this directory.
+
+Ignores differences in line endings, unless the test uses `--end_of_line`.
+**/
+import std.algorithm, std.array, std.conv, std.file, std.path, std.process;
+import std.stdio, std.string, std.typecons, std.range, std.uni;
+
+version (Windows)
+    enum dfmt = `..\bin\dfmt.exe`;
+else
+    enum dfmt = `../bin/dfmt`;
+
+int main()
+{
+    foreach (braceStyle; ["allman", "otbs", "knr"])
+        foreach (entry; dirEntries(".", "*.d", SpanMode.shallow).
+                            filter!(e => e.baseName(".d") != thisExePath.baseName(".exe")))
+        {
+            const source = entry.baseName;
+            const outFileName = buildPath(braceStyle, source ~ ".out");
+            const refFileName = buildPath(braceStyle, source ~ ".ref");
+            const argsFile = source.stripExtension ~ ".args";
+            const dfmtCommand = 
+                [dfmt, "--brace_style=" ~ braceStyle] ~ 
+                (argsFile.exists ? readText(argsFile).splitter!isWhite.filter!(a => a.length).array : []) ~
```

Comment:
```suggestion
                (argsFile.exists ? readText(argsFile).split : []) ~
```
the default split(ter) does it all for us (splits across multiple whitespace, ignores leading and trailing whitespace - will never produce an empty string in the result array)

Reply:
Yeah I was trying to accomplish that, but overread the details of the documentation. Thanks.